### DISCUSSION TITLE
Error when develop gapflow DroNet on AI-deck

### DISCUSSION THREAD

**Discussion by ChenMathilda** [bitcraze/discussions#59] [Q&A] [answered]
Hi,
I encountered some problems when deploying DroNet. When I used`make clean all run platform=gvsoc`
 under `/pulp-dronet-v2/gapflow/` to deploy the DroNet, I encountered this problem:https://pastebin.pl/view/9e1a8baf
```
# part of Error log
Error: Graph stacked tensor S16_Output, input tensor Output_1 is also defined as a graph local or graph argument
Execution aborted
```
I asked pulp-dronet on GitHub, but I didn't get any reply, so I can only ask here.
Is there anyone has solutions?
gap_sdk=3.8 python=3.7( I tried gap_sdk=3.9 but met more Errors )
export .pth from  `/pulp-dronet/pulp-dronet-v2/model/dronet_v2_gapflow_original_himax.pth ` and ` /pulp-dronet/pulp-dronet-v2/model/dronet_v2_gapflow_original.pth`

By the way, I post this question both forum and github.
Regards,
Mathilda

**Comment by knmcguire** [comment]
hi Mathilda!

Thanks for posting it here! The forum will be locked soon so this is the right place to ask. As I said, we haven't tried it ourselves yet with the new framework. 

Would you mind sharing the links of the issues you put on the pulp dronet github repo, for some context? Then people can see the linkage between this discussion and that repo.

**Reply by ChenMathilda** [reply]
> hi Mathilda!
> 
> Thanks for posting it here! The forum will be locked soon so this is the right place to ask. As I said, we haven't tried it ourselves yet with the new framework.
> 
> Would you mind sharing the links of the issues you put on the pulp dronet github repo, for some context? Then people can see the linkage between this discussion and that repo.

Hi 
this problem has been solved, thanks for you reply!
It was because the version of sdk(gap_sdk >=3.9.1)

**Reply by knmcguire** [reply]
ahh oke! Then they would need to change their documentation. I'll make an issue for it.

**Reply by knmcguire** [reply]
Btw, could you separately post this solution underneath in 'suggest an answer' of this discussion? That way, we can select your solution as the answer to this Q&A and you'll get kudos! I can also update my 'answer' to include your solution but then I'll be marked as most helpful, which seems to be a bit unfair 😉

**Comment by ChenMathilda** [comment, marked as answer]
Hi
this problem has been solved, thanks for you reply! It was because the version of sdk(gap_sdk >=3.9.1).
The official configuration is also gap_sdk >= 3.9.1. But the environment configuration of gap_sdk is too complicated, so I use the gap_sdk3.8 environment that I configured before, which caused this problem.
